cql3/query_processor.cc

```diff
@@ -348,6 +348,11 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, se
                             [] { return prepared_statements_cache::shard_stats().prepared_cache_evictions; },
                             sm::description("Counts the number of prepared statements cache entries evictions.")),
 
+                    sm::make_derive(
+                            "new_gen_on_cache_size_evictions",
+                            [] { return prepared_statements_cache::shard_stats().new_gen_on_cache_size_evictions; },
+                            sm::description("Counts the number of prepared statements cache entries evictions from the new generation partition due to a cache size restrictions. This is usually a sign of a cache pollution.")),
```

Comment:
Again, I don't think anybody will understand this explanation a week from now (or even today ;-)). The phrase "new generation partition" is opaque, and it's not clear why it's a problem or what it means ("cache pollution" is a very vague description of a problem).

Reply:
Please, suggests a better description/name.

Reply:
See my suggestion above - instead of mentioning "new generation" (a phrase which no user would understand), say something about "evicted after only been used once". 
Then, instead of "a sign of cache pollution" which is very vague, maybe say "a sign that the application is using prepared statements only once after preparation" (or something like that....).